### PR TITLE
core: update access token endpoints to work with policies

### PR DESCRIPTION
**Pull request by tessr**

### CODE REVIEW

**Comment by kr** [comment]
:eyes:

File: cmd/corectl/main.go [hunk 1]
```diff
@@ -237,6 +237,10 @@ func createToken(db *sql.DB, args []string) {
 		fatalln("error:", err)
 	}
 	fmt.Println(tok.Token)
+
+	if *flagNet {
+		fmt.Println("warning: the network flag is deprecated")
```
Comment (kr) [inline cmd/corectl/main.go]: This message should go to stderr. Prob `fmt.Fprintln(os.Stderr, ...)` is the way to go.

File: core/migrate/data.go [hunk 2]
```diff
@@ -60,4 +60,7 @@ var migrations = []migration{
 	{Name: `2017-04-13.0.query.block-transactions-count.sql`, SQL: `
 		ALTER TABLE annotated_txs ADD COLUMN block_tx_count INTEGER;
 	`},
+	{Name: `2017-04-17.0.core.null-token-type.sql`, SQL: `
+		ALTER TABLE access_tokens ALTER COLUMN type DROP NOT NULL;
```
Comment (kr) [inline core/migrate/data.go]: Don't we need to keep this so we can return the token's type if the user created it that way?

Comment (tessr) [inline core/migrate/data.go]: This just drops the constraint, not the column.

Comment (kr) [inline core/migrate/data.go]: Aha, my bad. :+1:

**Comment by kr** [comment]
LGTM

**Pull request merged**